dev/001_start.R

```diff
@@ -115,6 +115,7 @@ usethis::use_package("glue")
 usethis::use_dev_package("riskmetric")
 
 usethis::use_package("rstudioapi", type = "Suggests")
+usethis::use_package("png", type = "Suggests")
```

Comment:
I noticed that this is "added" to the project but not `renv.lock`. I'm assuming you used it to resize the pictures. Perhaps we should ignore the `dev/` folder with regards to `renv`.